server/map_rpc_server.go

```diff
@@ -205,6 +214,12 @@ func (t *TrillianMapServer) SetLeaves(ctx context.Context, req *trillian.SetMapL
 	}
 	ctx = trees.NewContext(ctx, tree)
 
+	if t.opts.UseSingleTransaction {
+		glog.Infof("EXP: Using experimental single-transaction mode")
```

Comment:
There's going to be a log of these logs -- is there a constructor we can move this log to?

Reply:
It was somewhat intentional, so the operator couldn't claim to have missed it :)
Moved to the c'tor.